packages/intl-messageformat/src/formatters.ts

```diff
@@ -86,7 +86,7 @@ function mergeLiteral<T>(
   }, [] as MessageFormatPart<T>[]);
 }
 
-function isFormatXMLElementFn<T>(
+export function isFormatXMLElementFn<T>(
```

Comment:
I exported this function for I need it in `react-intl`. I previously even declared almost exactly the same function in `react-intl`, but sharing is caring 😄